linode/manager
# M3-5641 & 5642: Update URL for invalid paths and cleanup routing

## Description

**M3-5641:**

Previously, the page would redirect to the first tab if the URL was invalid but the URL was not updated. This PR fixes that and also updates the tab-related functions to be consistent.

**M3-5642:**

Cleans up the routing so that it'll redirect to the landing pages and replace `RouteComponentProps` with `useRouteMatch` hook.

## How to test

Make sure none of the links have regressed and try breaking the URL from different pages.
